File: StructuralPattern/Facade/Python/Facade/GameFacade.py

from SubSystems import Plant, Zombie, GameEngine

class GameFacade:
    """
        Class facade for game
    """
    def __init__(self):
        self.gameEngine = GameEngine()
        self.plant = Plant()
        self.zombie = Zombie()
    
    def startGame(self):
        """
            Start game
        """
        self.gameEngine.startGame()
        self.plant.grow()
        self.zombie.spawn()
        self.plant.attackZombie()
        self.gameEngine.endGame()

File: StructuralPattern/Facade/Java/Facade/GameFacade.java

package Facade;

import SubSystems.*;

public class GameFacade {
    private Plant plant;
    private Zombie zombie;
    private GameEngine gameEngine;

    public GameFacade() {
        this.plant = new Plant();
        this.zombie = new Zombie();
        this.gameEngine = new GameEngine();
    }

    public void playGame() {
        gameEngine.startGame();
        plant.grow();
        zombie.spawn();
        plant.attackZombie();
        zombie.attackPlant();
        gameEngine.endGame();
    }
}